### DISCUSSION TITLE
does anyone know how to install the old version of tshock on the server, i want to install tshock version 4.3.16 on the server.pro site but there is no list of tshock for that version does anyone know how to ins

### DISCUSSION THREAD

**Discussion by Hex4gonForc3** [Pryaxis/TShock#3001] [General]
does anyone know how to install the old version of tshock on the server, i want to install tshock version 4.3.16 on the server.pro site but there is no list of tshock for that version does anyone know how to ins

**Comment by Arthri** [comment]
Version v4.3.16 is from almost 8 years ago. I would not be surprised if server.pro dropped support for it. I suggest opening a support ticket with server.pro as that is their platform

**Reply by Hex4gonForc3** [reply]
Do you know of a tshock terraria server hosting site that can use the tshock plugin version 4.3.16?

**Reply by Arthri** [reply]
No. Terraria's latest version can't join TShock from 8 years ago and I doubt any server still hosts and supports that version. Updating the plugin instead of downgrading TShock is a better course of action.

**Reply by Hex4gonForc3** [reply]
using the old version of terraria that corresponds to that old tshock version,I want to use the old version of tshock because I want to try the old version of the plugin files that have been used and see the server logs so I can see any problems using same new plugin

**Reply by sgkoishi** [reply]
TShock 4.3.16 is based on a completely different underlying system (design, architecture and runtime). It's meaningless as most plugins from that time are no longer maintained right now. You might find some but likely they are quite different - and maybe unrelated (but with the same/similar name and features).

People who can find the problem by reading old logs can probably fix the plugin without reading it. The fastest way is to read the error message and the code.

**Comment by ghost** [comment]
Here is your awnser: if you want this to be joined from multiple people in your network check your ip adress requrally using the mobile version of terraria as it will tell you your ip and if you want it to be you only make sure that nobody can join, and download the package from the packages section or something.
short awnser: host it on your pc
